<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manage</class>
 <widget class="QWidget" name="Manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>图书信息管理</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="6" column="0">
        <widget class="QPushButton" name="btn_all_2">
         <property name="text">
          <string>显示全表</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="tableView_3"/>
       </item>
       <item row="2" column="0">
        <widget class="QWidget" name="widget_7" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="btn_add">
            <property name="text">
             <string>添加图书</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_re_book">
            <property name="text">
             <string>修改选中图书</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_deletebook">
            <property name="text">
             <string>删除选中图书</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>查询字段</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_2">
             <item>
              <property name="text">
               <string>图书名称</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>作者</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>专业领域</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>查询信息</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btn_search_2">
           <property name="text">
            <string>确认查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>用户信息管理</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QPushButton" name="btn_add_2">
             <property name="text">
              <string>添加用户</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_re_account">
             <property name="text">
              <string>修改选中用户信息</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_deleteaccount">
             <property name="text">
              <string>删除选中用户信息</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>学号查询</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
           <item>
            <widget class="QPushButton" name="btn_search_3">
             <property name="text">
              <string>确认查询</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="btn_all_3">
             <property name="text">
              <string>显示全表</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QTableView" name="tableView_4"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>管理员信息管理</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView_5"/>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btn_addman">
             <property name="text">
              <string>添加管理员</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_reman">
             <property name="text">
              <string>修改选中管理员信息</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_deleteman">
             <property name="text">
              <string>删除选中管理员信息</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>管理员账号查询</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="btn_search">
             <property name="text">
              <string>确认查询</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btn_all">
           <property name="text">
            <string>显示全表</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btn_exit">
     <property name="text">
      <string>退出账号</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
